tests/ConfigurationTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+use Illuminate\Support\Facades\Config;
+use Spatie\Browsershot\Browsershot;
+use Spatie\LaravelPdf\Facades\Pdf;
+
+// Helper functions for testing Browsershot configuration
+function getPrivateProperty(object $object, string $property): mixed
```

Comment:
Could you use our spatie/invade package for this? It's specifically built for this purpose